src/app/public/src/modules/image/image.component.html

```diff
@@ -1,3 +1,7 @@
 <div class="stache-image">
   <img class="stache-image-object" [src]="imageSource" [alt]="imageAlt" />
+
+  <div *ngIf="description" class="stache-image-description">
+      <span><i>{{ description }}</i><span>
```

Comment:
Invalid closing tag for span here, causing build to fail

Reply:
I am not `100%` on the use of the `<i>` tags here, if they are appropriate.  Maybe @Blackbaud-ToddRoberts has more feedback, but I feel like we would be better served using a [SKY UX](https://developer.blackbaud.com/skyux/design/styles/typography) class for the span.  Maybe `sky-field-label` or `sky-deemphasized`?  Other thoughts about the text being left aligned vs center.

Reply:
Assuming this is a caption for an image? If so I'd recommend `sky-deemphasized` class rather than `<i>`, and left justify text.

Reply:
I originally looked at the typography page that you linked but didn't find a class that seemed to fit really well based off the description. If I had to pick between those two I'd say `sky-deemphasized`.

I think centering will create a cleaner look -- here's some examples:

![image](https://user-images.githubusercontent.com/35976987/45185501-6d62d700-b1f8-11e8-9803-8f56ffd18a62.png)